### ISSUE TITLE
Unchecked checkboxes are circular

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#25715]
### Steps to reproduce

1. Open one of the sort/appearance menus in the room list

#### What did you expect?

Checkboxes are boxes

#### What happened instead?

Checkboxes are circles, and therefore look like radio buttons

![Screenshot from 2023-07-04 13-43-06](https://github.com/vector-im/element-web/assets/48614497/eb02691a-6b27-46d9-805e-b7d77d63a1c4)

### Operating system

NixOS unstable

### Browser information

Firefox 114.0.1

### URL for webapp

develop.element.io

### Application version

Element version: 01a234aafaa4-react-2a7780052e60-js-5751df1288b3 Olm version: 3.2.14

### Homeserver

Not relevant

### Will you send logs?

No

**Comment by germain-gg** [comment]
Me and @andybalaam are not able to reproduce.
Could you check whether this still happens on your side?

**Comment by andybalaam** [comment]
I am unable to reproduce this with the latest code, using Firefox 115 on Ubuntu.

**Comment by andybalaam** [comment]
Going ahead with the release candidate even though this is still open, since it looks like it got fixed in the meantime.

**Comment by t3chguy** [comment]
![image](https://github.com/vector-im/element-web/assets/2403652/1a8f203c-29b1-44be-a7ea-22d1aab8af96)

@andybalaam I can reproduce on Firefox

**Comment by andybalaam** [comment]
OK, I can now reproduce on Firefox at both small and large font sizes:

![image](https://github.com/vector-im/element-web/assets/76812/590760c1-fa31-43a5-a573-ea262bf6d173)

![image](https://github.com/vector-im/element-web/assets/76812/af63f4e2-bebc-4600-91b4-914844245fb2)

**Comment by andybalaam** [comment]
(Especially small?)

**Comment by andybalaam** [comment]
I actually think the difference between checked and unchecked is a slight red herring, and the main issue is that they are both too round. Checked and unchecked have identical values of `height`, `width` and `border-radius`, so the slightly different rendering appears to be a Firefox quirk which is particularly noticeable when they are quite round.

**Comment by andybalaam** [comment]
At both small sizes:

![image](https://github.com/vector-im/element-web/assets/76812/3955771c-abb4-40de-bc03-bc362b3e841c)

and large sizes:

![image](https://github.com/vector-im/element-web/assets/76812/88ecbb83-9c67-4907-a267-00b42fdbcdef)

0.27rem looks similar to how it looked before, on Firefox.

**Comment by andybalaam** [comment]
The same problem is visible on Chromium, and the same fix (0.27rem) looks good:

![image](https://github.com/vector-im/element-web/assets/76812/f98ecf7e-92ba-4934-bf01-c2cc0e123558)

**Comment by andybalaam** [comment]
So I am going to attempt a fix that just changes this one number.
